### ISSUE TITLE
Start front end redesign

### ISSUE THREAD

**Issue by JayHelton** [JayHelton/icare#8]
This will need broken into separate tickets.

1.New pictures - too sterile - needs more lifestyle life pics and color

2.Toolbar - bigger and more bold

3.repair area - needs more space. 2 per row instead of 4 - change font and font color

4.Change find a location area - grey blue does not match well. More basic font - bigger

5. Conv., prof., lifetime warr., and price match area - get rid of spinners - way too wordy. Needs to be reworded and condensed. No one is really reading all of that

6. Map area -  looks very early 2000s - Different background picture - maybe find a better plug in

7. Maybe change map to display all results instead of having a crowded list of areas towards the bottom

8. device area - needs new pictures - displays item description twice, only needs one

9. buy device area - needs rearranged - move ad picture to the right and center align the text. needs fersher font

10. Sell a device area looks outdated - the drop down boxes for buyback price area is hard to read/grey on grey

11. Franchise area - better font - better picture/picture is not sized right - very crowded area - too in your face, has a requset info form and then an banner that asks if they want more info/too repetitive

12. School/business area - better pictures - also a little too crowded - font change

13. Blog - needs to keep up on content regularly or be cut - stale posts are not good for sites

14. Total site - font change - remove social media bar and find a better alternative.

**Comment by EmHelton** [comment]
Change the color of the buttons (no more green, orange, and pink)
